packages/components/src/box-control/input-controls.tsx

```diff
@@ -49,7 +55,9 @@ export default function BoxInputControls( {
 			 * Supports changing pair sides. For example, holding the ALT key
 			 * when changing the TOP will also update BOTTOM.
 			 */
-			if ( altKey ) {
+			// @ts-expect-error - event.altKey is only present when the change event was
+			// triggered by a keyboard event.
+			if ( event.altKey ) {
```

Comment:
Not sure it's a better solution, but we could avoid the `@ts-expect-error` by adding a typecast (and a runtime check for good measure?)

```suggestion
			if (
				event.nativeEvent instanceof KeyboardEvent &&
				( event as React.KeyboardEvent ).altKey
			) {
```

haven't tested it at runtime, though

Reply:
I'll add a TODO comment here because I think it should be addressed in a different layer, for example in how the upstream change handler passes events. c64c2a61